src/Microsoft.AspNetCore.Sockets.Http/HttpConnectionDispatcher.cs

```diff
@@ -420,6 +423,7 @@ private static string GetNegotiatePayload(string connectionId, HttpSocketOptions
             // Setup the connection state from the http context
             connection.User = context.User;
             connection.SetHttpContext(context);
+            logScope.ConnectionId = connection.ConnectionId;
```

Comment:
Add a comment explaining why it needs to be set (if it's null). Should we just check for null?